proto/options.proto

```diff
@@ -189,4 +189,5 @@ message Options {
   // 42 is reserved
   optional uint32 height_precision = 43 [default = 0];                    // Number of digits precision for heights returned
   optional bool roundabout_exits = 44 [default = true];                   // Whether to announce roundabout exit maneuvers
+  optional bool include_linear_references = 45;                           // Include linear references for graph edges returned in certain responses.
```

Comment:
i think we can drop the `include_` part of the name to be in line with other parameters like `roundabout_exits` and `guidance_views`

Reply:
Great idea.